lottie/src/main/java/com/airbnb/lottie/LottieAnimationView.java

```diff
@@ -118,6 +119,8 @@ private void init(@Nullable AttributeSet attrs) {
       addColorFilter(new SimpleColorFilter(ta.getColor(
           R.styleable.LottieAnimationView_lottie_colorFilter, Color.TRANSPARENT)));
     }
+
+    scale = ta.getFloat(R.styleable.LottieAnimationView_lottie_scale, Float.MAX_VALUE);
```

Comment:
Please bear with me here, are you suggesting something like this.

```
if(ta.hasValue(R.styleable.LottieAnimationView_lottie_scale)) {
  lottieDrawable.setScale(ta.getFloat(R.styleable.LottieAnimationView_lottie_scale, 1f));
}
```

When done loading `Composition` still try to set the drawable scale according to the screen size.
```
if (compWidth > screenWidth ||
        compHeight > screenHeight) {
      float xScale = screenWidth / (float) compWidth;
      float yScale = screenHeight / (float) compHeight;
      float maxScaleForScreen = Math.min(xScale, yScale);

      setScale(Math.min(maxScaleForScreen, lottieDrawable.getScale()));
...
}
```